sentinel/src/main/kotlin/com/infinum/sentinel/di/LibraryComponents.kt

```diff
@@ -42,6 +42,7 @@ internal object LibraryComponents {
             tools.filterIsInstance<CertificateTool>().firstOrNull()?.userCertificates.orEmpty(),
             onTriggered
         )
+        WorkManagerInitializer.init(domainComponent)
```

Comment:
Needs to be called here, else tools won't be visible in UI

Reply:
Why can't it be called from initialize method? Why the tools won't be visible in the UI?

Reply:
Okay, thanks for the explanation 👍